.vscode/launch.json

```
@@ -387,6 +387,39 @@
                 "/tmp/ota-image.bin"
             ],
             "cwd": "${workspaceFolder}"
+        },
+        {
+            "name": "LightingApp (Tizen)[remote]",
```

```suggestion
            "name": "LightingApp (Tizen) [remote]",
```
And maybe move this entry under the Linux one?